### DISCUSSION TITLE
two chia blockchains ??

### DISCUSSION THREAD

**Discussion by Isanderthul** [guydavis/machinaris#586] [Q&A]
what does it mean if there are two blockchains of the same type?
<img width="1284" alt="Screenshot 2022-02-26 at 11 29 33" src="https://user-images.githubusercontent.com/3693197/155837956-32b1f559-0a2a-49bb-9cde-187d6fda8610.png">

**Comment by Isanderthul** [comment]
I have done some tests and I am going to open an issue

**Comment by guydavis** [comment]
Hi, yes that's not right for sure!  

Please share a full screenshot of your Workers page.

Then try a reset of the [Machinaris database](https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-machinaris-database).
